docs/configuration/settings.rst

```diff
@@ -480,6 +480,28 @@ In order to replace it by another one:
     multiauth.policy.basicauth.use = myproject.authn.BasicAuthPolicy
 
 
+Mohawk request Auth
+:::::::::::::::::::
+
+`Mohawk <https://mohawk.readthedocs.io/en/latest/index.html>`_ implements the 
+`Hawk authorization scheme <https://github.com/hueniverse/hawk>`_, which 
+allows you to authenticate messages from your application backend to Kinto.
+In order to prevent replay attacks, nonce checking is enforced.  
+See `here <https://github.com/hueniverse/hawk#implementations>` for 
+some implementations of Hawk in different programming languages.
```

Comment:
Note: Why would someone be interested in checking these implementations from the settings docs?

Reply:
Maybe it's not necessary, but I included those references because the user will need to implement the hawk client.